plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/TrinoFileSystemCache.java

```diff
@@ -73,13 +75,15 @@ private TrinoFileSystemCache() {}
     public FileSystem get(URI uri, Configuration conf)
             throws IOException
     {
+        STATS.newGetCall();
         return getInternal(uri, conf, 0);
     }
 
     @Override
     public FileSystem getUnique(URI uri, Configuration conf)
             throws IOException
     {
+        STATS.newGetUniqueCall();
```

Comment:
This method seems to be called when `org.apache.hadoop.fs.FileSystem#newInstance(java.net.URI, org.apache.hadoop.conf.Configuration)` is being called.
However I haven't identified any call for this method in Trino.
@posulliv  while testing did you happen to identify a scenario where this counter gets actually incremented?

Reply:
if `getUnique(URI uri, Configuration conf)` is not called, it could be replaced with `throw...` but that's orthogonal to this PR. @findinpath feel free to experiment on your own